<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Olivier Trichet &lt;olivier@trichet.fr&gt;</author>
 <class>KNode::GroupSelection::BaseDialog</class>
 <widget class="KDialog" name="KNode::GroupSelection::BaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>714</height>
   </rect>
  </property>
  <widget class="QWidget" name="page" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>559</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="_2" stretch="0,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="mFilterContainer" native="true">
      <layout class="QHBoxLayout" name="_3" stretch="1,0,0,0">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="KFilterProxySearchLine" name="mSearchLine"/>
       </item>
       <item>
        <widget class="QCheckBox" name="mTreeviewCheckbox">
         <property name="text">
          <string comment="@option:check Display the list of newsgroups as a tree when enabled">Tree view</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mNewOnlyCheckbox">
         <property name="text">
          <string comment="@option:check Filter newly received group in the subscription dialog">New only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mSubscribedOnlyCheckbox">
         <property name="text">
          <string comment="@option:check Filter to only display subscribed groups">Subscribed</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="mViewSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeView" name="mGroupsView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="mChangeViewLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="1">
         <widget class="QTreeView" name="mChangeView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QWidget" name="mButtonContainer" native="true">
          <layout class="QVBoxLayout" name="_4" stretch="4,0,1,0,10">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <spacer name="mButtonSpacerTop">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>112</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="KPushButton" name="mAddChangeButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="mButtonSpacerMiddle">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>26</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="KPushButton" name="mRevertChangeButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="mButtonSpacerBottom">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>285</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDialog</class>
   <extends>QDialog</extends>
   <header>kdialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kfilterproxysearchline.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
